Fixed Kafka Exporter Grafana dashboard with N/A for empty numeric fields (#7215)
strimzi/strimzi-kafka-operator: documentation/modules/metrics/con_kafka-exporter-lag.adoc

The metrics data is used, for example, to help identify slow consumers.
Lag data is exposed as Prometheus metrics, which can then be presented in Grafana for analysis.

> The Grafana dashboard for Kafka Exporter, provided as an example, is available in the xref:ref-metrics-dashboards-{context}[Example Grafana dashboards] list.

Reviewer: ```suggestion
A Grafana dashboard for Kafka Exporter is one of a number of xref:ref-metrics-dashboards-{context}[example Grafana dashboards] provided by Strimzi.
```
